### PR TITLE
EG-784, joda.time to java.time

### PR DESCRIPTION
**Pull request by cmrhmrc**

### CODE REVIEW

File: app/services/AuditService.scala [hunk 1]
```diff
@@ -19,10 +19,11 @@ package services
 import config.AppConfig
 import javax.inject.{Inject, Singleton}
 import models.audit.AuditEvent
-import org.joda.time.DateTime
+// import java.time.{ZoneId, ZonedDateTime}
```
Comment (woollett-r) [inline app/services/AuditService.scala:22]: These imports can be removed.

File: project/AppDependencies.scala [hunk 2]
```diff
@@ -6,7 +6,7 @@ object AppDependencies {
     "uk.gov.hmrc"             %% "play-language"            % "4.3.0-play-26",
     "uk.gov.hmrc"             %% "bootstrap-play-26"        % "1.14.0",
     "uk.gov.hmrc"             %% "play-frontend-govuk"      % "0.49.0-play-26",
-    "uk.gov.hmrc"             %% "play-frontend-hmrc"       % "0.16.0-play-26",
+    "uk.gov.hmrc"             %% "play-frontend-hmrc"       % "0.17.0-play-26",
     "com.typesafe.play"       %% "play-json-joda"           % "2.6.13",
```
Comment (woollett-r) [inline project/AppDependencies.scala:10]: Is this library still required?

File: app/services/AuditService.scala (hunk 1 above)
Comment (cmrhmrc) [reply app/services/AuditService.scala:22]: done

File: project/AppDependencies.scala (hunk 2 above)
Comment (cmrhmrc) [reply project/AppDependencies.scala:10]: yes

Comment (cmrhmrc) [reply project/AppDependencies.scala]: Sorry joda, yes that can be removed. done

**Pull request merged**